src/main/java/net/sf/jabref/gui/keyboard/KeyBindingWrapper.java

```diff
@@ -0,0 +1,50 @@
+package net.sf.jabref.gui.keyboard;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import org.apache.commons.lang.WordUtils;
+
+public class KeyBindingWrapper {
+
+    private KeyBinding keyBinding;
+    private final SimpleStringProperty bind;
+    private final KeyBindingCategory category;
+
+
+    public KeyBindingWrapper(KeyBinding keyBinding, String bind, KeyBindingCategory category) {
```

Comment:
All items in a `TreeTableView` are fom the same type, also the expandable ones. Since those are no real KeyBindings but rather only their category, I created the this wrapper class as view model to include the categories as well as the keyBindings.